docs/website/views/_includes/landing/article.njk

```diff
@@ -3,7 +3,7 @@
     <a class="article-link" href="{{ article.link }}" target="_blank">
       <h5 class="article-item-title mb-2">{{ article.title }}</h5>
       {% if isFirst %}
-        <esl-image class="article-item-img img-container img-container-16-9 " data-src="{{ article.img }}" lazy alt="{{ article.title }}" mode="cover"></esl-image>
+        <esl-image class="article-item-img img-container" data-src="{{ article.img }}" lazy alt="{{ article.title }}" mode="fit"></esl-image>
```

Comment:
Should be `img-container-4-3` container
Otherwise we will have a layout shift on page load